frontend/benefit/applicant/src/components/applications/pageContent/PageContent.tsx

```diff
@@ -0,0 +1,59 @@
+import StepperActions from 'benefit/applicant/components/applications/forms/application/stepperActions/StepperActions';
+import {
+  StyledHeaderItem,
+  StyledPageHeader,
+  StyledPageHeading,
+} from 'benefit/applicant/components/applications/styled';
+import { DynamicFormStepComponentProps } from 'benefit/applicant/types/common';
+import dynamic from 'next/dynamic';
+import React, { ComponentType } from 'react';
+import Container from 'shared/components/container/Container';
+import Stepper from 'shared/components/stepper/Stepper';
+
+import { useComponent } from './extended';
+
+const PageContent = (): React.ReactElement => {
+  const {
+    handleSubmit,
+    handleBack,
+    t,
+    steps,
+    currentStep,
+    hasBack,
+    hasNext,
+  } = useComponent();
+
+  const DynamicFormStepComponent: ComponentType<DynamicFormStepComponentProps> = dynamic(
```

Comment:
Also This one is simple: https://codesandbox.io/s/github/devrnt/react-use-wizard/tree/main/examples/nextjs?file=/pages/index.js:1050-1054
You dont have to give hasBack and hasNext params. It does not preserve the step number if reloaded as the solution in previous comment

Reply:
i don't like the idea of the routing in wizard to be honest. that's overcomplication. preserving the state does not give you any advantage. You have to store the current step in backend anyway, on refresh, on edit application, on new application, etc. I didnt actually know that that the dynamic import does not work with literal strings. But in anyway, it can be easily replaced

Reply:
It is enough for now just to fix the dynamic import. The state of the wizard must be saved somewhere if you want to preserve it after refresh. Number of the step in the url is easiest way imho. It can also be a queryparam or hash I guess...